src/player_data.h

```diff
@@ -92,20 +92,27 @@ enum PlayerVictoryState {
     VicS_State3,
 };
 
-enum PlayerField454Val { // These could be Cursor states...?
-    P454_Unkn0 = 0,
-    P454_Unkn1,
-    P454_Unkn2,
-    P454_Unkn3,
+enum PlayerCursorStates {
+    CSt_DefaultArrow = 0, // Default - Arrow Cursor
+    CSt_PickAxe, // Dig - Pickake cursor
+    CSt_Key, // Lock/Unlock Door - Key cursor
+    CSt_PowerHand, // Power Hand?
 };
 
 enum PlayerField3Flags {
     Pf3F_None   = 0x00,
-    Pf3F_Unkn01 = 0x01,
-    Pf3F_Unkn02 = 0x02,
-    Pf3F_Unkn04 = 0x04,
-    Pf3F_Unkn08 = 0x08,
-    Pf3F_Unkn10 = 0x10,
+    Pf3F_nothing_under_power_hand = 0x01, // Chosen subtile has nothing to interact with with the Power Hand (no creature to slap etc) (But the power hand is active)
+    Pf3F_chosen_subtile_is_high = 0x02, // Chosen subtile is at ceiling height (dirt/rock/wall etc)
+    Pf3F_alternate_palette_is_active = 0x04, // something to do with palette swapping/toggling
+    // [used in PaletteSetPlayerPalette...]
+    // [used during creature fade - if Pf3F_Unkn04 is OFF set red palette, otherwise set engine palette]
+    // [used to toggle on the blue_pal/engine_pal of a creature if it is affected by freeze spell]
+    Pf3F_lightning_palette_is_active = 0x08, // something to do with palette swapping/toggling
+    //[turned on with lightning_palette]
+    //[turned off within lightning functions when engine_palette is applied]
+    //[if the game is paused -  if Pf3F_Unkn08 is "ON" then engine_palette is applied and Pf3F_Unkn08 is turned "OFF"]
```

Comment:
You should rename Pf3F_Unkn08 here and 5 lines above

Reply:
These comments will be tidied up before the end of this PR